app/init.php

```diff
@@ -110,7 +110,7 @@
 const APP_USER_ACCCESS = 24 * 60 * 60; // 24 hours
 const APP_CACHE_UPDATE = 24 * 60 * 60; // 24 hours
 const APP_CACHE_BUSTER = 514;
```

Comment:
lets also update the cache buster

Reply:
We'll leave this since there's no update to the console